pkg/web/templates/partials/transaction/transaction_preview.html

```diff
@@ -9,7 +9,19 @@ <h1 class="text-xl font-bold text-gray-900 lg:text-2xl">
           </h1>
         </div>
         <!-- TODO: Make buttons same width -->
-        <form hx-post="/v1/transactions/send" hx-ext="json-enc" method="post">
+        <form 
+          _="on htmx:afterRequest
+              if event.detail.successful
+                remove @open from #preview_envelope
+                then reset() the #secure-envelope-form
+                then reload() the location of the window
+                then go to top of the body" 
+          hx-post="/v1/transactions/send" 
+          hx-target="#secure-envelope-form" 
+          hx-swap="none" 
```

Comment:
`hx-swap` has been set to none after running into an issue where the selects weren't displaying after the swap. This is due to using a library to make the selects searchable, so refreshing the page is the simplest solution for now.

Reply:
Ah interesting; does the library you're using initialize based on class or a data- tag?